resources/data/relive_index.json

```diff
@@ -0,0 +1,90 @@
+[
+   {
+      "index_url" : "//cdn.c3voc.de/relive/wevsclimatecrisis/index.json",
+      "media_conference_id" : "wevsclimatecrisis",
+      "project" : "wevsclimatecrisis",
+      "updated_at" : 1601127243
+   },
+   {
+      "index_url" : "//cdn.c3voc.de/relive/rc3/index.json",
+      "media_conference_id" : "rc3",
+      "project" : "rc3",
+      "updated_at" : 1609279570
+   },
+   {
+      "index_url" : "//cdn.c3voc.de/relive/pw20/index.json",
+      "media_conference_id" : "pw20",
+      "project" : "pw20",
+      "updated_at" : 1609279575
+   },
```

Comment:
You can probably reduce the test data to a small subset of the "live" files to concentrate on the stuff you are testing..